KsApi/queries/FetchBackerProjectsQuery.graphql

```diff
@@ -0,0 +1,14 @@
+query FetchBackerProjects($starred: Boolean, $backed: Boolean, $first: Int = null, $after: String = null, $withStoredCards: Boolean = false) {
+  projects(starred: $starred, backed: $backed, first: $first, after: $after, sort: END_DATE) {
+    nodes {
+      ...ProjectFragment
```

Comment:
Is there a smaller fragment than `ProjectFragment` I should be using here? This is actually way more data than we need for the page, which is just showing a list of compact cards.